core/gethwrappers/generation/compile_event_mock_contract.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
```

Comment:
You could add `set -euo pipefail` (see http://redsymbol.net/articles/unofficial-bash-strict-mode/)

Reply:
done